backend/src/prompts/templates/intent-system.j2

```diff
@@ -0,0 +1,110 @@
+You are an expert in determining the intent behind a user's question. 
+
+Your task is to accurately comprehend the intentions behind the current question. 
+The question can be composed of different intents and may depend on the context provided by the previous question and its response.
+
+- You must evaluate whether the current question is directly related to or dependent on the information from the previous interaction.
```

Comment:
There seems to be a discrepancy between words "question" and "query" they seem to mean the same thing to it would be best to use only one work to describe this. - probably "question"